src/main/java/com/conveyal/datatools/editor/datastore/FeedTx.java

```diff
@@ -165,30 +166,31 @@ public void buildSecondaryIndices () {
         tripCountByPatternAndCalendar = getMap("tripCountByPatternAndCalendar");
         Bind.histogram(trips, tripCountByPatternAndCalendar, (tripId, trip) -> new Tuple2(trip.patternId, trip.calendarId));
 
+        // Removed scheduleExceptionCountByDate due to corruption issues.
         // getting schedule exception map appears to be causing issues for some feeds
         // The names of the code writers have been changed to protect the innocent.
-        try {
-            scheduleExceptionCountByDate = getMap("scheduleExceptionCountByDate");
-        } catch (RuntimeException e1) {
-            LOG.error("Error getting scheduleExceptionCountByDate map. Getting a new one.");
-            int count = 0;
-            final int NEW_MAP_LIMIT = 100;
-            while (true) {
-                try {
-                    scheduleExceptionCountByDate = getMap("scheduleExceptionCountByDateMapDBIsTheWORST" + count);
-                } catch (RuntimeException e2) {
-                    LOG.error("Error getting {} scheduleExceptionCountByDateMapDBIsTheWORST map. Getting a new one.", count);
-                    count++;
-                    if (count > NEW_MAP_LIMIT) {
-                        LOG.error("Cannot create new map. Reached limit of {}", NEW_MAP_LIMIT);
-                        throw e2;
-                    }
-                    continue;
-                }
-                break;
-            }
-        }
-        BindUtils.multiHistogram(exceptions, scheduleExceptionCountByDate, (id, ex) -> ex.dates.toArray(new LocalDate[ex.dates.size()]));
+//        try {
```

Comment:
If we're not going to be using this we should just delete it rather than commenting it out. The existing code is preserved in version control so no need to keep it as a comment block unless our intent is to rewrite or refactor it soon.